extensions/amp-story/0.1/amp-story.js

```diff
@@ -1094,6 +1073,19 @@ export class AmpStory extends AMP.BaseElement {
   }
 
 
+  /**
+   * @return {!Promise<boolean>}
+   * @private
+   */
+  hasBookend_() {
+    if (!this.isDesktop_()) {
+      return Promise.resolve(true);
+    }
```

Comment:
Nope. Bookend always will show up on mobile since it always includes the replay button. UX on desktop was updated so that the forward button allows for replay when reaching end-of-story, which is on the last page when no related articles are present.

Reply:
Can you add this in a comment?

Reply:
Done.